corehq/apps/app_manager/models.py

```diff
@@ -5284,8 +5284,6 @@ class Application(ApplicationBase, TranslationMixin, HQMediaMixin):
     use_grid_menus = BooleanProperty(default=False)
     grid_form_menus = StringProperty(default='none',
                                      choices=['none', 'all', 'some'])
-    mobile_ucr_sync_interval = IntegerProperty()
```

Comment:
I've always been hesitant to remove properties like this. If we ever make a new property that has the same name, then these apps could have a value that means something different in the future.

Reply:
True, though I don't think it's worth keeping the code just for that (especially in this case since there are only 3 apps that I know of that have values set for this property).